services/maven-central/maven-central.tester.js

```diff
@@ -22,18 +22,13 @@ t.create('latest 0.8 version')
 
 t.create('inexistent artifact')
   .get('/v/inexistent-group-id/inexistent-artifact-id.json')
-  .expectBadge({ label: 'maven-central', message: 'invalid' })
-
-t.create('connection error')
-  .get('/v/com.github.fabriziocucci/yacl4j.json')
-  .networkOff()
-  .expectBadge({ label: 'maven-central', message: 'inaccessible' })
+  .expectBadge({ label: 'maven-central', message: 'not found' })
 
 t.create('xml parsing error')
```

Comment:
I think we can probably get rid of this test altogether too, as it's testing functionality in the Base class, which has its own tests elsewhere